<h1><div>Review of Homology-directed repair of a defective glabrous gene in Arabidopsis with Cas9-based gene targeting</div></h1><div><h1 data-label="463319" class="ltx_title_section"><b>Homology-directed repair of a defective glabrous gene in Arabidopsis with Cas9-based gene targeting &nbsp;</b></h1><h2 data-label="720048" class="ltx_title_subsection"><b>[<b>Florian&nbsp;Hahn,&nbsp;Marion&nbsp;Eisenhut,&nbsp;Otho&nbsp;Mantegazza,&nbsp;Andreas P.M.&nbsp;Weber, January 5, 2018, BioRxiv</b>] </b></h2><h3 data-label="236037" class="ltx_title_subsubsection"><b>[<a href="https://doi.org/10.1101/243675">https://doi.org/10.1101/243675</a>]</b></h3><h1 data-label="463319" class="ltx_title_section"></h1><div><b>Overview and take-home messages:</b></div><div></div><div>&nbsp;&nbsp;&nbsp;&nbsp;Hahn et al. have compared the efficiencies of two different methods that have been previously reported to enhance the frequency of homologous recombination in plants. The paper has focused on testing a viral replicon system with two different enzymes, nuclease and nickase, as well as an&nbsp;<i>in planta</i> gene targeting (IPGT) system in&nbsp;<i> Arabidopsis thaliana</i>. Interestingly, authors have chosen GLABROUS1 (GL1), a regulator of trichome formation, as a visual marker to detect Cas9 activity and therefore homologous recombination. A 10 bp deletion in the coding region of&nbsp;<i>GL1</i> gene produces plants devoid of trichomes. Out of the two methods&nbsp;<i>in planta</i> gene targeting approach successfully restored trichome formation in less than 0.2% of ~2,500 plants screened, whereas the method based on viral replicon machinery did not manage to restore trichome formation at all. This manuscript is of high quality, experiments are well designed and executed. However, there are some concerns that could be addressed in the next preprint or print version. Below are some feedback and suggestions that we hope will improve the manuscript.</div></div><div><div><b>Positive feedback:</b></div><div></div><div>&nbsp;&nbsp;&nbsp;&nbsp;The constructs that authors have made for their study (Fig. 1) are well thought out. Using a different 10 base-pair long nucleotide sequence, yet encoding same amino acids, within the constructs&nbsp;was a clever idea to prevent a possible pollen contamination from wild-type plants. Subsequently, Figure 2 shows the working principle of the constructs shown in Figure 1. The design of Figure 2. did a good job of helping the reader to visualize and differentiate between methods of enhancing HT availability.&nbsp;</div></div><div><div><b>Major concerns:</b></div><div></div><div>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Figure 1 is well constructed and presented, but it was slightly unclear in the beginning about why the repair template was 10 bp long with CTGCCGTTTA as its sequence. We feel, including the Supplemental Figure 1.1 A into Figure 1 of the main paper would be very helpful in understanding the steps taken to generate&nbsp;<i>gl1&nbsp;</i>CRISPR lines and the reason for choosing this repair template. Nonetheless, we believe it was a clever idea. Figure 3B show the random somatic events occurred in the T2 generation. However, it was not clear what this figure wanted to tell us.</div><div>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;In the chromatogram shown in Figure 4B, authors have claimed that peaks occurring at the site of Cas9 cleavage indicate a biallelic or chimeric mutation. However, we believe this chromatogram simply shows a heterozygous plant. In order it to be considered chimeric or biallelic the chromatogram should have shown three peaks, not double peaks.&nbsp;</div><div>&nbsp; &nbsp; &nbsp; &nbsp; Furthermore,&nbsp; using the loss of trichomes as a visual marker first sounded a good idea, however, after taking the number of plants needed for screening into account we think the authors could have chosen a different visual marker that is easier to distinguish.</div><div>&nbsp; &nbsp; &nbsp; &nbsp; Lastly, we would like to know whether the authors have tried and tested a pIPTG-Nick&nbsp;construct? Since they have managed to achieve repair only in pIPGT-Nuc construct, we think it is very crucial to test the same construct with nickase enzyme.&nbsp;</div></div><div><div><b>Minor concerns:</b></div></div><div><div>&nbsp;&nbsp;&nbsp;&nbsp;The authors indicate in Figure 4 that some plants appeared to have fully restored trichome formation in the T3 generation. Sequencing results in panel B show that the non-glaborous plants are heterozygous as opposed to their chimeric parents. In the Results paragraph for Figure 4, it would be beneficial to indicate that the plants are heterozygous for more clarity. In addition, for panel 4B, it would help the reader if the color of the nucleotide letters on the X-axis matched the color of the peaks.&nbsp;</div><div>&nbsp;&nbsp;&nbsp;&nbsp;Figure 3A, left is referenced in the text (lines 285-288)to be a T3 plant, however, the figure legend states that all plants shown in the figure are T2 plants. The authors state that plants transformed with pVIR-Nick are shown in Figure 3A (line 274), nevertheless, plants transformed with pIPGT-Nuc and pVIR-Nuc are shown. In the text and in Figure 3B it is stated that there are 42 clones but when the different clones displayed in Figure 3B are added together the total number adds up to 41.</div><div></div><div></div></div><div><h1 data-label="993054" class="ltx_title_section"></h1><div></div></div><div><h1 data-label="993054" class="ltx_title_section"></h1><div></div></div><div><h1 data-label="993054" class="ltx_title_section"></h1><div></div></div>