\documentclass[10pt]{article}

\usepackage{fullpage}
\usepackage{setspace}
\usepackage{parskip}
\usepackage{titlesec}
\usepackage{placeins}
\usepackage{xcolor}
\usepackage{breakcites}
\usepackage{lineno}





\PassOptionsToPackage{hyphens}{url}
\usepackage[colorlinks = true,
            linkcolor = blue,
            urlcolor  = blue,
            citecolor = blue,
            anchorcolor = blue]{hyperref}
\usepackage{etoolbox}
\makeatletter
\patchcmd\@combinedblfloats{\box\@outputbox}{\unvbox\@outputbox}{}{%
  \errmessage{\noexpand\@combinedblfloats could not be patched}%
}%
\makeatother


\usepackage[round]{natbib}
\let\cite\citep




\renewenvironment{abstract}
  {{\bfseries\noindent{\abstractname}\par\nobreak}\footnotesize}
  {\bigskip}

\renewenvironment{quote}
  {\begin{tabular}{|p{13cm}}}
  {\end{tabular}}

\titlespacing{\section}{0pt}{*3}{*1}
\titlespacing{\subsection}{0pt}{*2}{*0.5}
\titlespacing{\subsubsection}{0pt}{*1.5}{0pt}


\usepackage{authblk}


\usepackage{graphicx}
\usepackage[space]{grffile}
\usepackage{latexsym}
\usepackage{textcomp}
\usepackage{longtable}
\usepackage{tabulary}
\usepackage{booktabs,array,multirow}
\usepackage{amsfonts,amsmath,amssymb}
\providecommand\citet{\cite}
\providecommand\citep{\cite}
\providecommand\citealt{\cite}
% You can conditionalize code for latexml or normal latex using this.
\newif\iflatexml\latexmlfalse
\providecommand{\tightlist}{\setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}%

\AtBeginDocument{\DeclareGraphicsExtensions{.pdf,.PDF,.eps,.EPS,.png,.PNG,.tif,.TIF,.jpg,.JPG,.jpeg,.JPEG}}

\usepackage[utf8]{inputenc}
\usepackage[english]{babel}








\begin{document}

\title{Review of Homology-directed repair of a defective glabrous gene in
Arabidopsis with Cas9-based gene targeting}



\author[1]{Elsbeth Walker}%
\author[1]{David Chan Rodriguez}%
\author[1]{Stavroula Fili}%
\author[1]{Ahmet Bakirbas}%
\author[1]{Rebecca Brennan}%
\author[1]{Ahmed Ali}%
\author[1]{Rakesh Krishna Kumar}%
\author[1]{Miriam Hernandez Romero}%
\author[1]{Maura Zimmermann}%
\author[1]{Gurpal Singh}%
\affil[1]{University of Massachusetts Amherst}%


\vspace{-1em}



  \date{\today}


\begingroup
\let\center\flushleft
\let\endcenter\endflushleft
\maketitle
\endgroup









\section*{Homology-directed repair of a defective glabrous gene in
Arabidopsis with Cas9-based gene targeting
~}

{\label{463319}}

\subsection*{\texorpdfstring{{[}\textbf{Florian~Hahn,~Marion~Eisenhut,~Otho~Mantegazza,~Andreas
P.M.~Weber, January 5, 2018,
BioRxiv}{]}}{{[}Florian~Hahn,~Marion~Eisenhut,~Otho~Mantegazza,~Andreas P.M.~Weber, January 5, 2018, BioRxiv{]}}}

{\label{720048}}

\subsubsection*{\texorpdfstring{{[}\url{https://doi.org/10.1101/243675}{]}}{{[}https://doi.org/10.1101/243675{]}}}

{\label{236037}}

\section*{}

{\label{463319}}

\textbf{Overview and take-home messages:}

\par\null

~~~~Hahn et al. have compared the efficiencies of two different methods
that have been previously reported to enhance the frequency of
homologous recombination in plants. The paper has focused on testing a
viral replicon system with two different enzymes, nuclease and nickase,
as well as an~\emph{in planta} gene targeting (IPGT) system in~
\emph{Arabidopsis thaliana}. Interestingly, authors have chosen
GLABROUS1 (GL1), a regulator of trichome formation, as a visual marker
to detect Cas9 activity and therefore homologous recombination. A 10 bp
deletion in the coding region of~\emph{GL1} gene produces plants devoid
of trichomes. Out of the two methods~\emph{in planta} gene targeting
approach successfully restored trichome formation in less than 0.2\% of
\textasciitilde{}2,500 plants screened, whereas the method based on
viral replicon machinery did not manage to restore trichome formation at
all. This manuscript is of high quality, experiments are well designed
and executed. However, there are some concerns that could be addressed
in the next preprint or print version. Below are some feedback and
suggestions that we hope will improve the manuscript.

\textbf{Positive feedback:}

\par\null

~~~~The constructs that authors have made for their study (Fig. 1) are
well thought out. Using a different 10 base-pair long nucleotide
sequence, yet encoding same amino acids, within the constructs~was a
clever idea to prevent a possible pollen contamination from wild-type
plants. Subsequently, Figure 2 shows the working principle of the
constructs shown in Figure 1. The design of Figure 2. did a good job of
helping the reader to visualize and differentiate between methods of
enhancing HT availability.~

\textbf{Major concerns:}

\par\null

~~~~~~~~Figure 1 is well constructed and presented, but it was slightly
unclear in the beginning about why the repair template was 10 bp long
with CTGCCGTTTA as its sequence. We feel, including the Supplemental
Figure 1.1 A into Figure 1 of the main paper would be very helpful in
understanding the steps taken to generate~\emph{gl1~}CRISPR lines and
the reason for choosing this repair template. Nonetheless, we believe it
was a clever idea. Figure 3B show the random somatic events occurred in
the T2 generation. However, it was not clear what this figure wanted to
tell us.

~~~~~~~~In the chromatogram shown in Figure 4B, authors have claimed
that peaks occurring at the site of Cas9 cleavage indicate a biallelic
or chimeric mutation. However, we believe this chromatogram simply shows
a heterozygous plant. In order it to be considered chimeric or biallelic
the chromatogram should have shown three peaks, not double peaks.~

~ ~ ~ ~ Furthermore,~ using the loss of trichomes as a visual marker
first sounded a good idea, however, after taking the number of plants
needed for screening into account we think the authors could have chosen
a different visual marker that is easier to distinguish.

~ ~ ~ ~ Lastly, we would like to know whether the authors have tried and
tested a pIPTG-Nick~construct? Since they have managed to achieve repair
only in pIPGT-Nuc construct, we think it is very crucial to test the
same construct with nickase enzyme.~

\textbf{Minor concerns:}

~~~~The authors indicate in Figure 4 that some plants appeared to have
fully restored trichome formation in the T3 generation. Sequencing
results in panel B show that the non-glaborous plants are heterozygous
as opposed to their chimeric parents. In the Results paragraph for
Figure 4, it would be beneficial to indicate that the plants are
heterozygous for more clarity. In addition, for panel 4B, it would help
the reader if the color of the nucleotide letters on the X-axis matched
the color of the peaks.~

~~~~Figure 3A, left is referenced in the text (lines 285-288)to be a T3
plant, however, the figure legend states that all plants shown in the
figure are T2 plants. The authors state that plants transformed with
pVIR-Nick are shown in Figure 3A (line 274), nevertheless, plants
transformed with pIPGT-Nuc and pVIR-Nuc are shown. In the text and in
Figure 3B it is stated that there are 42 clones but when the different
clones displayed in Figure 3B are added together the total number adds
up to 41.

\par\null\par\null

\selectlanguage{english}
\FloatBarrier
\end{document}

