<div>In addition to efficiently targeting SOX2 (34% increase in methylation after 3 days, half of it persistent after 9 days, and 2.5 fold decrease in mRNA) in MCF7 breast cancer cells, this fusion protein was able to bind 25142 off-target sites. This serendipitous finding enabled the genome-wide study of induced 5mC in regulatory regions using whole genome bisulfite sequencing (WGBS). Differential methylation was indeed found in more than 10K regions (DMRs) (by the way, it would be good to define DMR early in the text, in terms of size and number of CpG sites). It was not evident how the fusion protein selects its targets, considering that many were not CpG-rich regions, and why there is little overlap between protein binding sites and DMRs (35% as assessed by ChIP). Although the authors illustrate this in many ways, it may have been of interest to know the actual genomic distance between DMRs and the nearest ChIP peak.</div><div></div><div>Despite concerns about target affinity, this fusion protein efficiently induced 5mC (hypomethylation was almost absent). In addition, such induced 5mC was only partially associated with gene expression. In my view, one very interesting point of this work was that the authors included a "resting" condition, where 3-day doxycicline induction of&nbsp;ZF-DNMT3A was followed by 9 additional days w/o doxycicline. Using this strategy, they were able to show that most inducible 5mC is reversible, and especially at CpG-rich loci. Was there any difference in ZF-DNMT3A binding (or proximity) between "stable" and "reversible" DMRs?</div><div></div><div>DNA methylation can be lost trough passive dilution (i.e. DNMT1 impairment) or active removal (i.e. by TET dyoxigenases). Kinetics experiments (Fig 4B) are compatible with, approximately, half reduction of 5mC after each cell division, and therefore with the first option. The authors seem to rule out this possibility using cell cycle inhibitors. However, the evidence for this is less convincing. For example, the cell cycle profiles shown in Fig S4A do not indicate a strong synchronization. It is probably difficult to design such experiment, but a stronger effect may have been achieved with a mitotic arrest (e.g. nocodazol). In addition, it would have been relevant to see a positive control of passive demethylation, such as the one that may be obtained with 5-azacytidine. Finally, although the authors  indirectly show TET activity (by 5hmC profiling), functional silencing of TETs would be a more relevant way to demonstrate that 5mC loss is an active process in their model. In my view, the presented data is not enough evidence to conclude that "induced DNA methylation is actively demethylated". As a side note, WGBS does not distinguish 5mC from 5hmC, so it is possible that 5mC loss after dox removal is underestimated. Indeed,  ZF-DNMT3A would be a useful tool to also study the kinetics of other 5mC derivatives, such as 5fC and 5caC.</div><div></div><div>Additional information is provided in this study, in terms of active histone marks and initiated Pol II binding to DMRs. Together with the expression levels, this indicates that forced DNA methylation does not necessarily interfere with transcription. It is not clear if RNAseq data was of enough depth to also rule out differences at the level of isoform expression or alternative splicing events. This work also features a nice RNA-FISH strategy, used to rule out methylation mixed-response populations.</div><div></div><div>Other points include: RNAseq in Fig 1B is not described, there is no mention about association with H3K27ac (although data is shown), no description of treatment time on the legend to Fig S4, and no indication of time point for evaluating 5hmC kinetics. There is no clear indication of the number of replicates used in the WGBS experiment. Although technical validation is always important, if no or few replicates are included, a clear validation strategy should be stated.&nbsp;</div>