\documentclass[10pt]{article}

\usepackage{fullpage}
\usepackage{setspace}
\usepackage{parskip}
\usepackage{titlesec}
\usepackage{placeins}
\usepackage{xcolor}
\usepackage{breakcites}
\usepackage{lineno}





\PassOptionsToPackage{hyphens}{url}
\usepackage[colorlinks = true,
            linkcolor = blue,
            urlcolor  = blue,
            citecolor = blue,
            anchorcolor = blue]{hyperref}
\usepackage{etoolbox}
\makeatletter
\patchcmd\@combinedblfloats{\box\@outputbox}{\unvbox\@outputbox}{}{%
  \errmessage{\noexpand\@combinedblfloats could not be patched}%
}%
\makeatother


\usepackage[round]{natbib}
\let\cite\citep




\renewenvironment{abstract}
  {{\bfseries\noindent{\abstractname}\par\nobreak}\footnotesize}
  {\bigskip}

\renewenvironment{quote}
  {\begin{tabular}{|p{13cm}}}
  {\end{tabular}}

\titlespacing{\section}{0pt}{*3}{*1}
\titlespacing{\subsection}{0pt}{*2}{*0.5}
\titlespacing{\subsubsection}{0pt}{*1.5}{0pt}


\usepackage{authblk}


\usepackage{graphicx}
\usepackage[space]{grffile}
\usepackage{latexsym}
\usepackage{textcomp}
\usepackage{longtable}
\usepackage{tabulary}
\usepackage{booktabs,array,multirow}
\usepackage{amsfonts,amsmath,amssymb}
\providecommand\citet{\cite}
\providecommand\citep{\cite}
\providecommand\citealt{\cite}
% You can conditionalize code for latexml or normal latex using this.
\newif\iflatexml\latexmlfalse
\providecommand{\tightlist}{\setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}%

\AtBeginDocument{\DeclareGraphicsExtensions{.pdf,.PDF,.eps,.EPS,.png,.PNG,.tif,.TIF,.jpg,.JPG,.jpeg,.JPEG}}

\usepackage[utf8]{inputenc}
\usepackage[english]{babel}








\begin{document}

\title{PREreview of ``Frequent lack of repressive capacity of promoter DNA
methylation identified through genome-wide epigenomic manipulation''}



\author[1]{Hector Hernandez-Vargas}%
\affil[1]{Cancer Research Center of Lyon (CRCL)}%


\vspace{-1em}



  \date{\today}


\begingroup
\let\center\flushleft
\let\endcenter\endflushleft
\maketitle
\endgroup





\selectlanguage{english}
\begin{abstract}
This is a review of the preprint\textbf{~``Frequent lack of repressive
capacity of promoter DNA methylation identified through genome-wide
epigenomic manipulation''}~by Ethan Edward Ford,~ Matthew R. Grimmer,~
Sabine Stolzenburg,~ Ozren Bogdanovic,~ Alex de Mendoza,~ Peggy J.
Farnham,~ Pilar Blancafort, and~ Ryan Lister.

The preprint was originally posted
on~\href{https://www.biorxiv.org/}{bioRxiv}~on September 20, 2017
(DOI:~\url{https://doi.org/10.1101/170506}).~%
\end{abstract}%






\section*{Review}

{\label{874460}}\par\null

Despite being heavily studied, we still don't understand the exact
consequences of DNA methylation on gene expression. There is a positive
association between cytosine methylation (5mC) in gene bodies and their
transcriptional status. On the other hand, regulatory regions such as
promoters or enhancers tend to be negatively correlated with gene
expression, with CG density playing an important role. Moreover,
physiological processes such as imprinting, or pathologies such as
cancer, indicate that CpG island (CGI) methylation is strongly
associated with gene silencing. It remains to be answered whether 5mC is
a cause or a consequence of such silencing.

\par\null

The authors of this~\href{https://doi.org/10.1101/170506}{preprint}
approach this interesting question, particularly at the level of gene
promoters. To this end, they took advantage of the off-target effects of
a zinc finger (ZF)-DNMT3A fusion protein (shown in their Fig 1A, below)
originally designed to target a GC-rich 18 bp sequence in the SOX2
promoter.\selectlanguage{english}
\begin{figure*}[h!]
\begin{center}
\includegraphics[width=0.70\columnwidth]{figures/mass/Untitled-1}

\end{center}
\end{figure*}

In addition to efficiently targeting SOX2 (34\% increase in methylation
after 3 days, half of it persistent after 9 days, and 2.5 fold decrease
in mRNA) in MCF7 breast cancer cells, this fusion protein was able to
bind 25142 off-target sites. This serendipitous finding enabled the
genome-wide study of induced 5mC in regulatory regions using whole
genome bisulfite sequencing (WGBS). Differential methylation was indeed
found in more than 10K regions (DMRs) (by the way, it would be good to
define DMR early in the text, in terms of size and number of CpG sites).
It was not evident how the fusion protein selects its targets,
considering that many were not CpG-rich regions, and why there is little
overlap between protein binding sites and DMRs (35\% as assessed by
ChIP). Although the authors illustrate this in many ways, it may have
been of interest to know the actual genomic distance between DMRs and
the nearest ChIP peak.

\par\null

Despite concerns about target affinity, this fusion protein efficiently
induced 5mC (hypomethylation was almost absent). In addition, such
induced 5mC was only partially associated with gene expression. In my
view, one very interesting point of this work was that the authors
included a ``resting'' condition, where 3-day doxycicline induction
of~ZF-DNMT3A was followed by 9 additional days w/o doxycicline. Using
this strategy, they were able to show that most inducible 5mC is
reversible, and especially at CpG-rich loci. Was there any difference in
ZF-DNMT3A binding (or proximity) between ``stable'' and ``reversible''
DMRs?

\par\null

DNA methylation can be lost trough passive dilution (i.e. DNMT1
impairment) or active removal (i.e. by TET dyoxigenases). Kinetics
experiments (Fig 4B) are compatible with, approximately, half reduction
of 5mC after each cell division, and therefore with the first option.
The authors seem to rule out this possibility using cell cycle
inhibitors. However, the evidence for this is less convincing. For
example, the cell cycle profiles shown in Fig S4A do not indicate a
strong synchronization. It is probably difficult to design such
experiment, but a stronger effect may have been achieved with a mitotic
arrest (e.g. nocodazol). In addition, it would have been relevant to see
a positive control of passive demethylation, such as the one that may be
obtained with 5-azacytidine. Finally, although the authors indirectly
show TET activity (by 5hmC profiling), functional silencing of TETs
would be a more relevant way to demonstrate that 5mC loss is an active
process in their model. In my view, the presented data is not enough
evidence to conclude that ``induced DNA methylation is actively
demethylated''. As a side note, WGBS does not distinguish 5mC from 5hmC,
so it is possible that 5mC loss after dox removal is underestimated.
Indeed, ZF-DNMT3A would be a useful tool to also study the kinetics of
other 5mC derivatives, such as 5fC and 5caC.

\par\null

Additional information is provided in this study, in terms of active
histone marks and initiated Pol II binding to DMRs. Together with the
expression levels, this indicates that forced DNA methylation does not
necessarily interfere with transcription. It is not clear if RNAseq data
was of enough depth to also rule out differences at the level of isoform
expression or alternative splicing events. This work also features a
nice RNA-FISH strategy, used to rule out methylation mixed-response
populations.

\par\null

Other points include: RNAseq in Fig 1B is not described, there is no
mention about association with H3K27ac (although data is shown), no
description of treatment time on the legend to Fig S4, and no indication
of time point for evaluating 5hmC kinetics. There is no clear indication
of the number of replicates used in the WGBS experiment. Although
technical validation is always important, if no or few replicates are
included, a clear validation strategy should be stated.~

Overall, this is a very interesting work, clearly presented and
illustrated with attractive and informative visualizations. Although
more work is needed to understand how 5mC stability is achieved, the
authors correctly conclude that in their model DNA methylation itself is
frequently insufficient to transcriptionally repress promoters. This
information may prove highly useful for designing methylome-editing
strategies.

\par\null

Thanks again for posting this interesting work as a preprint,

Hector Hernandez-Vargas

\selectlanguage{english}
\FloatBarrier
\end{document}

