<h1 data-label="457509" class="ltx_title_section"><span class="math ltx_Math v1">\(\)</span>Comments</h1><div></div><div><b>Phylogenetic analyses. </b>The authors report that RPS5 and PBR1 are not orthologous based on phylogenetic analysis (Figure 4), and as a consequence conclude that the capacity to recognize AvrPphB activity is an example of convergent evolution in Arabidopsis and barley. However, the phylogenetic trees shown in Fig. 4a and 4b&nbsp; lack the broader context necessary to determine the relatedness of the depicted NLRs. Phylogenetic analyses that Include a wider diversity of Arabidopsis and barley NLRs are necessary to conclusively define the evolutionary relationship between AtRPS5 and PBR1.</div><div></div><div><b>Cell-death assays.</b> To study PBR1 role in AvrPphB recognition, the authors carried out cell-death assays (Figure 6) by transiently expressing both the NLR and the effector in N. benthamiana. However, the presence of an endogenous PBS1 homolog in these assays limit the level of mechanistic insight that can be gained from these experiments. We suggest using a PBS1-silenced or loss-of-function background to obtain unambiguous results.&nbsp; </div><div></div><div><b>Figure 2A.</b> In the experiments depicted in Figure 2A, what was the rationale behind employing a Bayesian phylogenetic analysis as opposed to other statistical methods? In general, it would be good to justify or to mention whether other phylogenetic methods have yielded similar findings.</div><div></div><div><b>Figure 3A.</b> In the experiments depicted in Figure 3A, what was the rationale behind including the barley line with the Low Chlorosis (LC) response phenotype to AvrPphB in the GWAS as opposed to just using non-AvrPphB responsive lines? </div><div><span class="math ltx_Math v1">\(\)</span></div><h1 data-label="924286" class="ltx_title_section">Reviewers</h1><div></div><div>Mauricio P Contreras, Erin K. Zess, and Sophien Kamoun. The Sainsbury Laboratory, Norwich Research Park, Norwich, UK.</div><div></div>