\documentclass[10pt]{article}

\usepackage{fullpage}
\usepackage{setspace}
\usepackage{parskip}
\usepackage{titlesec}
\usepackage{placeins}
\usepackage{xcolor}
\usepackage{breakcites}
\usepackage{lineno}





\PassOptionsToPackage{hyphens}{url}
\usepackage[colorlinks = true,
            linkcolor = blue,
            urlcolor  = blue,
            citecolor = blue,
            anchorcolor = blue]{hyperref}
\usepackage{etoolbox}
\makeatletter
\patchcmd\@combinedblfloats{\box\@outputbox}{\unvbox\@outputbox}{}{%
  \errmessage{\noexpand\@combinedblfloats could not be patched}%
}%
\makeatother


\usepackage[round]{natbib}
\let\cite\citep




\renewenvironment{abstract}
  {{\bfseries\noindent{\abstractname}\par\nobreak}\footnotesize}
  {\bigskip}

\renewenvironment{quote}
  {\begin{tabular}{|p{13cm}}}
  {\end{tabular}}

\titlespacing{\section}{0pt}{*3}{*1}
\titlespacing{\subsection}{0pt}{*2}{*0.5}
\titlespacing{\subsubsection}{0pt}{*1.5}{0pt}


\usepackage{authblk}


\usepackage{graphicx}
\usepackage[space]{grffile}
\usepackage{latexsym}
\usepackage{textcomp}
\usepackage{longtable}
\usepackage{tabulary}
\usepackage{booktabs,array,multirow}
\usepackage{amsfonts,amsmath,amssymb}
\providecommand\citet{\cite}
\providecommand\citep{\cite}
\providecommand\citealt{\cite}
% You can conditionalize code for latexml or normal latex using this.
\newif\iflatexml\latexmlfalse
\providecommand{\tightlist}{\setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}%

\AtBeginDocument{\DeclareGraphicsExtensions{.pdf,.PDF,.eps,.EPS,.png,.PNG,.tif,.TIF,.jpg,.JPG,.jpeg,.JPEG}}

\usepackage[utf8]{inputenc}
\usepackage[english]{babel}








\begin{document}

\title{OIST E\&E PREreview Journal Club, ``Frequency of disturbance alters
diversity, function, and underlying assembly mechanisms of complex
bacterial communities''}



\author[1]{maggi brisbin}%
\author[1]{Jigyasa Arora}%
\author[1]{Crystal Clitheroe}%
\author[1]{Fabien C. Y. Benureau}%
\author[1]{Julian Katzke}%
\author[1]{Paula Villa Martin}%
\author[1]{Yuka Suzuki}%
\author[1]{Samuel Ross}%
\author[1]{Stefano Pascarelli}%
\author[1]{Alicia Tovar}%
\affil[1]{Okinawa Institute of Science and Technology Graduate School}%


\vspace{-1em}



  \date{\today}


\begingroup
\let\center\flushleft
\let\endcenter\endflushleft
\maketitle
\endgroup









\section*{Frequency of disturbance alters diversity, function, and
underlying assembly mechanisms of complex bacterial
communities}

{\label{463319}}

\subsection*{Ezequiel~Santillan,~Hari~Seshan,~Florentin~Constancias,~Daniela
I.~Drautz-Moses,~Stefan~Wuertz, May 4th 2018,
bioRxiv}

{\label{720048}}

\subsubsection*{\texorpdfstring{{[}doi:~\url{https://doi.org/10.1101/313585}{]}}{{[}doi:~https://doi.org/10.1101/313585{]}}}

{\label{236037}}

Understanding the effects of disturbance on ecosystem function and
diversity has many potential applications in microbial ecology and human
disease biology. In this paper, the authors tackled the long-standing
question of how different disturbance frequencies affect bacterial
community diversity and function. To do so, activated-sludge communities
within laboratory-scale microcosms were exposed to toxic 3-chloroaniline
(3-CA) at varying frequencies. Ecosystem function and community
diversity were measured weekly by measuring biomass and organic carbon,
ammonia, and toxin removal as proxies for ecosystem function and T-RFLP
16S rRNA gene fingerprinting and shotgun metagenomics were performed to
examine variation in bacterial diversity and community composition. This
work is an excellent example of integrating genomic and functional
analysis, thereby providing a more thorough understanding of the effects
of disturbance frequency on microbial community diversity and function.
Interestingly, both genetic methods yielded similar results, suggesting
that the less expensive gene fingerprinting method could be sufficient
when sequencing resources are limited. We particularly commend the use
of multiple alpha-diversity measurements and the inclusion of
abundance-related indices, which are less method dependent and allow
results to be compared between studies. Ultimately, the authors propose
the ``Intermediate Stochasticity Hypothesis,'' which suggests that
stochastic processes produce higher diversity assemblages at
intermediate disturbance frequencies while deterministic processes
produce lower diversity assemblages at low and high disturbance
frequencies. Overall, this paper is a fascinating and substantial
contribution to microbial ecology. There are, however, a few issues that
we feel could be improved in future versions of the manuscript.~

~

\textbf{Major concerns:}

\par\null

This comment is unique to the preprint. The manuscript references
multiple figures available in the supplementary materials, but these
materials were not made available as part of the preprint. This hindered
our ability to understand the fine points of the experiments. We
encourage the authors to upload the supplementary materials to bioRxiv.~

\par\null

1. Figure 2 is an integral figure to the manuscript because it showcases
the effects of 3-CA disturbance frequencies on community performance,
namely organic carbon and toxin removal (plots A, C)~ and nitrification
products (plots B, D). In the Materials and Methods section (lines
353-356), the authors state that these parameters were measure weekly,
which leads to the assumption that data is available for days 7, 15, 21,
and 35, even though only the data from days 7 and 35 are included in the
figure (is there T0 data?). The results section refers to supplementary
figures S2 and S3 in addition to Figure 2, so these supplementals may
portray the data of interest. However, since these data are so important
to the overall conclusions, we believe it should be available in the
main text. One way to accomplish this could be to have one plot per
variable with time on the x-axis and different colors for each
disturbance frequency. The number of plots could be reduced by not
including Volatile Suspended Solids (VSS) results in the main text.~

\par\null

In Figure 2A, the COD removal and 3-CA removal is not monotonously
decreasing relative to the disturbance frequency (specifically, level 2
and 4). We figured that this was due to the number of days since the
disturbance being different for each disturbance frequency at
measurement time on day 7. We encourage the authors to mention and
explain this in the text, as this was a puzzling feature of the results
for us for some time. It also calls into question the appropriateness of
the weekly measurements, especially given that some disturbance level
will be highly correlated to this rhythm of measurement (level 1
disturbance will always happen on the same day of the measurements,
while level 2 and others will drift).~

\par\null

2. Along with disturbance frequency, varied intensity and duration of
disturbance and differing sampling frequencies (e.g- data collection
every two days or ~bi-weekly, larger spread of intermediate disturbance
levels) might produce a different pattern of microbial community
diversity and function. Questions we can ask are: would the system reach
the observed IDH pattern at an early stage? Would the intermediate
levels still follow the IDH model? We would be very interested in the
authors opinions on these topics, perhaps in the discussion section.~

\textbf{Minor concerns:}

\par\null

1.~When discussing disturbance frequencies and `levels' throughout the
manuscript, consistency of language is key. These different treatment
`levels' are misleading if described as disturbance levels since this
description can be interpreted as disturbance intensity if not read
carefully. Clarity of language surrounding disturbance manipulation is
really important for specific understanding and placing the study in the
wider context of studies of disturbance. We suggest changing `levels' to
frequency/ies' throughout.~

\par\null

2. We suggest including T0 data in the NMDS plot in Figure 1B. However,
we were not able~to understand why two different ordination methods were
used in Figure 1 and suggest using only one method (NMDS or PCoA). The
plot could be combined into one, if color represents disturbance
frequency and shape represents time.~

\par\null

3. The frequency of measurements implies sampling with replacement (but
this was not mentioned in the methods section), we would like to see a
description of how replacement was achieved and discussion of what the
impacts of replacement may have been. We are also interested in the
implications of scaling up the microcosm size and varying initial
conditions to reproduce and expand the experimental~design for further
work testing the new model.~

\par\null

4. Since the Results section appears before the Materials and Methods
section in this manuscript, we suggest writing out the full names of
abbreviated terms in the Results section so that readers can read
sections in the order they appear and know what abbreviations
represent.~

\par\null

5. The symbols and colors chosen for the figures made it difficult to
interpret the figures in many cases. For example, in figure 3, L4 and L6
are both represented by light grey squares that are very difficult to
discern in the legend and are not visible in the plot (the plot may have
been changed without updating the legend?). To make it easier to
interpret figures, we suggest choosing one color scheme for all figures
and keeping the colors for each disturbance level consistent throughout
all figures. Additionally, if points are overlapping, we suggest
increasing the alpha (transparency) of the points. Finally, it would be
significantly easier and quicker to interpret the figures if legends
were incorporated into the figures.~~

\par\null

6. Although, at several points in the paper, the authors reference the
softwares used, and sometimes, the corresponding parameters, we would
encourage the authors to share both the raw data (the performance
indicators in addition to the raw sequences, which are available on
NCBI) and the accompanying code used to analyze it (github or similar
site). In some cases the citations are missing for the relevant packages
or software. Sharing the code would shed light on the details of some
procedures that are not made explicit in the manuscript, and increase
the reproducibility of the experiment.

\par\null

This comment is unique to the preprint. The layout of the preprint,
which we understand is likely the result of the requirements of a
submission format, makes understanding the figures rather challenging.
For future preprint submissions, we encourage the authors to consider
associating the figures with their titles and captions, and to put the
figures inline, close to the relevant parts of the text.

\par\null

Overall, it was a great pleasure reading this interesting and exciting
work and we are extremely grateful that the authors posted it as a
preprint on bioRxiv. We sincerely hope that our comments are useful to
the authors and we look forward to reading the final version when it is
published.~

\par\null

Very best wishes,

The OIST Ecology and Evolution Preprint Journal Club

\selectlanguage{english}
\FloatBarrier
\end{document}

