<h1><div>Live streamed Journal Club on "Host-parasite interaction explains variation in prevalence of avian haemosporidians at the community level"</div></h1><div><div>This is a  review of the bioRxiv preprint "Host-parasite interaction explains variation in prevalence of avian haemosporidians at the community level" by Luz Garcia-Longoria, Alfonso Marzal, Florentino de Lope, and Laszlo Garamszegi. DOI:&nbsp;<a href="https://doi.org/10.1101/432260">https://doi.org/10.1101/432260</a>&nbsp;This review was compiled from discussion points raised during a PREreview live-streamed Ecology preprint journal club as part of Open Access Week,&nbsp;October 24, 2018.&nbsp;The event details can be found&nbsp;<a href="https://prereview.org/users/153686/articles/325778-prereview-plos-open-access-week-preprint-journal-club-information" target="_blank">here</a>&nbsp;and the collaborative Etherpad showing all the journal club notes can be found&nbsp;<a href="https://etherpad.net/p/EcologyLiveStreamedPREJC" target="_blank">here</a>.&nbsp;</div><div></div><div>In addition to those named as authors above, the participants who wished to be acknowledged for their contributions to this review are as follows:&nbsp;Dariusz Murakowski,&nbsp;Irene Ramos,&nbsp;Dena Emmerson,&nbsp;Ad√©la&nbsp; Nacer,&nbsp;Asar Khan, and Daniela Saderi.</div></div><div><h1 data-label="759186" class="ltx_title_section">Summary:&nbsp;</h1><div>In this paper the authors tackle the question of which host variables (phylogenetic, seasonal, or host-pathogen-specific) affect parasite prevalence&nbsp; and specialization in various species of bird.&nbsp;&nbsp;</div><div></div><h1 data-label="488681" class="ltx_title_section">Overview:</h1><div>This study used an impressive number of individuals from over a nine year period to address an interesting question that can have implications not only in ecology  but also in human health and how it will interact with climate change. However, some of the participants found the narrative a bit confusing and thought a clearer description of the objectives in the introduction would be helpful. Additional clarification was also requested in the methods. In particular, the molecular methods were not clear particularly on how new lineages were determined with the PCR approach and how the authors dealt with co-infections.&nbsp;</div><div></div><h1 data-label="657196" class="ltx_title_section">Major point:</h1><div></div><ul><li>One of the main questions that was raised during the discussion was how the 5 new parasite lineages were defined. The manuscript would benefit from more justification for the molecular approaches used to identify and define these lineages. For the PCR and sequencing for example, it was not clear why a single nucleotide difference equates to a new lineage. PCR amplification can cause single nucleotide changes and also this lineage identification is based on a single gene. Even if cytochrome b is conserved enough to allow this inference, single nucleotide changes could be a PCR artifact. Further discussion around these points would be significantly improve the manuscript.</li></ul><div></div><h1 data-label="657196" class="ltx_title_section">Minor points:</h1><ul><li>Table 3 was identified as confusing by a number of participants who also suggested that perhaps a more detailed caption could alleviate this. Also,  the authors should expand on what it means to be rescaled, and what the biological meaning is.</li><li>&nbsp;It would be helpful to include the phylogenetic trees (even in the supplemental) or combined with figure 1 </li><li>For figure 1, in addition to the above suggestion, it would be useful to include (n=) next to host species along x axis, and&nbsp; add "Parasite" to the "lineage" legend</li><li>It would be interesting to see an additional figure that shows a plot across time: how prevalence changed over the 9 years of the study&nbsp; (even average prevalence)</li><li>ggplot palates are not colour blind friendly and suggest using shapes or a colour blind friendly palate like&nbsp;<a href="https://www.nature.com/articles/nmeth.1618">https://www.nature.com/articles/nmeth.1618</a></li><li>It would be helpful to mention how the authors accounted for potential co-infections. This is something that can be pretty common to be infected by more than one lineage, which may complicate the analyses. The authors should comment on this in the discussion, if not elsewhere</li></ul><h1 data-label="520965" class="ltx_title_section">Typos: </h1><ul><li><div>Line 137: "were" --&gt; was?</div></li></ul><div></div></div><div><ul><li><div>Line 93: "become" --&gt; becomes</div></li></ul></div><div><ul><li><div>Line 228: explicitly mention which two additional factors affected variance in prevalence</div></li></ul></div><div><ul><li><div>Line 228-230: "Thus" sounds out of place; perhaps combine sentences with a colon. (At least, I think breeding season and parasite lineage are the two factors. Cleaning up the language here would clarify.)</div></li></ul></div><div><ul><li><div>Line 247: "change" --&gt; changes?</div></li></ul></div><div><ul><li><div>Typo line 52: remove "mainly"</div></li></ul></div><div><ul><li><div>Typo line 158: should be: relied "on"</div></li></ul></div><div><ul><li><div>Line 227: "where..." is sentence fragment</div></li></ul></div><div><ul><li><div>Line 281: change "this study" to "that study"</div></li></ul></div><div><ul><li><div>Typo: line 330, "analysed" should be "analysis"</div></li></ul></div><div><ul><li><div>Typo: line 101 Showed --&gt; shown</div></li></ul></div><div><div></div><div></div><div></div></div><div><div></div></div><div><div></div></div>