\documentclass[10pt]{article}

\usepackage{fullpage}
\usepackage{setspace}
\usepackage{parskip}
\usepackage{titlesec}
\usepackage{placeins}
\usepackage{xcolor}
\usepackage{breakcites}
\usepackage{lineno}





\PassOptionsToPackage{hyphens}{url}
\usepackage[colorlinks = true,
            linkcolor = blue,
            urlcolor  = blue,
            citecolor = blue,
            anchorcolor = blue]{hyperref}
\usepackage{etoolbox}
\makeatletter
\patchcmd\@combinedblfloats{\box\@outputbox}{\unvbox\@outputbox}{}{%
  \errmessage{\noexpand\@combinedblfloats could not be patched}%
}%
\makeatother


\usepackage[round]{natbib}
\let\cite\citep




\renewenvironment{abstract}
  {{\bfseries\noindent{\abstractname}\par\nobreak}\footnotesize}
  {\bigskip}

\renewenvironment{quote}
  {\begin{tabular}{|p{13cm}}}
  {\end{tabular}}

\titlespacing{\section}{0pt}{*3}{*1}
\titlespacing{\subsection}{0pt}{*2}{*0.5}
\titlespacing{\subsubsection}{0pt}{*1.5}{0pt}


\usepackage{authblk}


\usepackage{graphicx}
\usepackage[space]{grffile}
\usepackage{latexsym}
\usepackage{textcomp}
\usepackage{longtable}
\usepackage{tabulary}
\usepackage{booktabs,array,multirow}
\usepackage{amsfonts,amsmath,amssymb}
\providecommand\citet{\cite}
\providecommand\citep{\cite}
\providecommand\citealt{\cite}
% You can conditionalize code for latexml or normal latex using this.
\newif\iflatexml\latexmlfalse
\providecommand{\tightlist}{\setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}%

\AtBeginDocument{\DeclareGraphicsExtensions{.pdf,.PDF,.eps,.EPS,.png,.PNG,.tif,.TIF,.jpg,.JPG,.jpeg,.JPEG}}

\usepackage[utf8]{inputenc}
\usepackage[ngerman,english]{babel}








\begin{document}

\title{Live streamed Journal Club on ``Host-parasite interaction explains
variation in prevalence of avian haemosporidians at the community
level''}



\author[1]{Monica Granados}%
\author[1]{Samantha Hindle}%
\affil[1]{PREreview}%


\vspace{-1em}



  \date{\today}


\begingroup
\let\center\flushleft
\let\endcenter\endflushleft
\maketitle
\endgroup





\selectlanguage{english}
\begin{abstract}
This is a review of the bioRxiv preprint ``Host-parasite interaction
explains variation in prevalence of avian haemosporidians at the
community level'' by Luz Garcia-Longoria, Alfonso Marzal, Florentino de
Lope, and Laszlo Garamszegi.
DOI:~\url{https://doi.org/10.1101/432260}~This review was compiled from
discussion points raised during a PREreview live-streamed Ecology
preprint journal club as part of Open Access Week,~October 24, 2018.~The
event details can be
found~\href{https://prereview.org/users/153686/articles/325778-prereview-plos-open-access-week-preprint-journal-club-information}{here}~and
the collaborative Etherpad showing all the journal club notes can be
found~\href{https://etherpad.net/p/EcologyLiveStreamedPREJC}{here}.~

\par\null

In addition to those named as authors above, the participants who wished
to be acknowledged for their contributions to this review are as
follows:~Dariusz Murakowski,~Irene Ramos,~Dena Emmerson,~Ad\selectlanguage{ngerman}Ã©la~
Nacer,~Asar Khan, and Daniela Saderi.%
\end{abstract}\selectlanguage{ngerman}%




\section*{Summary:~}

{\label{759186}}

In this paper the authors tackle the question of which host variables
(phylogenetic, seasonal, or host-pathogen-specific) affect parasite
prevalence~ and specialization in various species of bird.~~

\par\null

\section*{Overview:}

{\label{488681}}

This study used an impressive number of individuals from over a nine
year period to address an interesting question that can have
implications not only in ecology but also in human health and how it
will interact with climate change. However, some of the participants
found the narrative a bit confusing and thought a clearer description of
the objectives in the introduction would be helpful. Additional
clarification was also requested in the methods. In particular, the
molecular methods were not clear particularly on how new lineages were
determined with the PCR approach and how the authors dealt with
co-infections.~

\par\null

\section*{Major point:}

{\label{657196}}\par\null

\begin{itemize}
\tightlist
\item
  One of the main questions that was raised during the discussion was
  how the 5 new parasite lineages were defined. The manuscript would
  benefit from more justification for the molecular approaches used to
  identify and define these lineages. For the PCR and sequencing for
  example, it was not clear why a single nucleotide difference equates
  to a new lineage. PCR amplification can cause single nucleotide
  changes and also this lineage identification is based on a single
  gene. Even if cytochrome b is conserved enough to allow this
  inference, single nucleotide changes could be a PCR artifact. Further
  discussion around these points would be significantly improve the
  manuscript.
\end{itemize}

\par\null

\section*{Minor points:}

{\label{657196}}

\begin{itemize}
\tightlist
\item
  Table 3 was identified as confusing by a number of participants who
  also suggested that perhaps a more detailed caption could alleviate
  this. Also, the authors should expand on what it means to be rescaled,
  and what the biological meaning is.
\item
  ~It would be helpful to include the phylogenetic trees (even in the
  supplemental) or combined with figure 1
\item
  For figure 1, in addition to the above suggestion, it would be useful
  to include (n=) next to host species along x axis, and~ add
  ``Parasite'' to the ``lineage'' legend
\item
  It would be interesting to see an additional figure that shows a plot
  across time: how prevalence changed over the 9 years of the study~
  (even average prevalence)
\item
  ggplot palates are not colour blind friendly and suggest using shapes
  or a colour blind friendly palate
  like~\url{https://www.nature.com/articles/nmeth.1618}
\item
  It would be helpful to mention how the authors accounted for potential
  co-infections. This is something that can be pretty common to be
  infected by more than one lineage, which may complicate the analyses.
  The authors should comment on this in the discussion, if not elsewhere
\end{itemize}

\section*{Typos:}

{\label{520965}}

\begin{itemize}
\tightlist
\item
  Line 137: ``were'' --\textgreater{} was?
\end{itemize}

\par\null

\begin{itemize}
\tightlist
\item
  Line 93: ``become'' --\textgreater{} becomes
\end{itemize}

\begin{itemize}
\tightlist
\item
  Line 228: explicitly mention which two additional factors affected
  variance in prevalence
\end{itemize}

\begin{itemize}
\tightlist
\item
  Line 228-230: ``Thus'' sounds out of place; perhaps combine sentences
  with a colon. (At least, I think breeding season and parasite lineage
  are the two factors. Cleaning up the language here would clarify.)
\end{itemize}

\begin{itemize}
\tightlist
\item
  Line 247: ``change'' --\textgreater{} changes?
\end{itemize}

\begin{itemize}
\tightlist
\item
  Typo line 52: remove ``mainly''
\end{itemize}

\begin{itemize}
\tightlist
\item
  Typo line 158: should be: relied ``on''
\end{itemize}

\begin{itemize}
\tightlist
\item
  Line 227: ``where\ldots{}'' is sentence fragment
\end{itemize}

\begin{itemize}
\tightlist
\item
  Line 281: change ``this study'' to ``that study''
\end{itemize}

\begin{itemize}
\tightlist
\item
  Typo: line 330, ``analysed'' should be ``analysis''
\end{itemize}

\begin{itemize}
\tightlist
\item
  Typo: line 101 Showed --\textgreater{} shown
\end{itemize}

\selectlanguage{english}
\FloatBarrier
\end{document}

