<h1 data-label="457509" class="ltx_title_section"><span class="math ltx_Math v1">\(\)</span>Findings and comments</h1><div></div><h2 data-label="408285" class="ltx_title_subsection"><b>(1)&nbsp;&nbsp;Overall comments</b></h2><div>The lack of functional analyses with pathogens and/or effectors and the fact that the study does not identify a differential phenotype between the reported subfamilies, do not support the statements about functional diversification.</div><div></div><div>CC domain differences are proposed to drive the diversification of the MLA receptor. However, the phylogeny of the conserved NB domain does not fully recapitulate this diversification.</div><div></div><div>The manuscript would benefit from a more thorough functional assessment of a range of CC domains. Only one CC domain from the new subclade, which is also the closest homolog to Sr50, has been assayed. Testing more members across the newly identified subfamily will help to draw a general conclusion about the new subclade. </div><div></div><div>The manuscript proposes a possible diversification in signalling capabilities, however, the chimera experiments of <cite class="ltx_cite raw v1">\cite{Jordan2011}</cite> are not consistent with this hypothesis and suggest conserved signalling capacities.</div><div></div><h2 data-label="390222" class="ltx_title_subsection"><b>(2)&nbsp;&nbsp;Comments on structural predictions</b></h2><div>The 21st residue of the MLA family CC domains is generally occupied by an aspartate or a glutamate whereas it is a glycine in Sr33, and it is suggested in the manuscript that this may account for the reported differences in the structures—Sr33 was described as a four-helix bundle by&nbsp;<cite class="ltx_cite raw v1">\cite{Casey2016}</cite>, and MLA10 as a helix-loop-helix in an obligate dimer by <cite class="ltx_cite raw v1">\cite{Maekawa2011}</cite>. The&nbsp;<cite class="ltx_cite raw v1">\cite{Casey2016}</cite> paper has shown that the CC domains of Sr33, MLA10 and Rx all maintain the same oligomeric state and four-helix bundle fold in solution, and this is supported by biophysical analyses of recombinantly produced protein. As such, the current debate on the CC domain structure is not centred around differences in their tertiary structures. What is unknown is whether the MLA10 CC domain dimeric helix-loop-helix structure represents an alternative quaternary conformation, for example a post-activation conformation. To this end, the manuscript does not address the “alternative activation state hypothesis” (discussed by&nbsp;<cite class="ltx_cite raw v1">\cite{27803318}</cite>), rather the text implies that the tertiary structures of Sr33 and MLA10 are different. To support the statement that Sr33 and MLA10 CC domains maintain different tertiary structures, the authors applied secondary structure prediction with PSIPRED and protein stability modelling with the STRUM web-server. However, published biochemical and biophysical data demonstrating the structural similarity of the Sr33 and MLA10 proteins in solution are not fully considered.&nbsp;</div><div></div><div>Secondary structure predictions of the MLA10 and Sr33 CC domains were stated to be performed with the first 40 amino acids “for simplicity”. This is problematic as secondary structure prediction using PSIPRED can vary depending on the length of the sequence submitted. Indeed, when the first 160 amino acids of MLA10 and Sr33 are submitted to PSIPRED, the observed differences in the "looped vs helical" regions of the first 40 residues of Sr33 and MLA10 reported in this manuscript are no longer apparent. Considering that the expression of the 1-160 region of the CC domains (or equivalent) triggered cell death in planta (Figure 4a), we believe this region to be more appropriate for secondary structure predictions.</div><div></div><div>It was hypothesized that the presence of the glycine at the 21st residue in Sr33 is the determinant of the “structural differences between MLA10 and Sr33”, and subsequently used the STRUM server (structure-based prediction of protein stability changes upon single-point mutation), to predict whether reciprocal mutations of polymorphisms between MLA10 and Sr33 (Sr33 V20T and Sr33 G21E; MLA10 T20V and MLA10 E21G) would be sufficient to destabilise the MLA10 and Sr33 structures, respectively. The results of STRUM suggest the MLA10 T20V and MLA10 E21G would likely destabilise the MLA10 structure, however the reciprocal mutations in Sr33 would have no effect. There are several questions that this analysis raises listed below. </div><div></div><ul><li>What was the structure used to model the destabilisation? The original MLA10 structure (3QFL) could be compared to the four-helix bundle fold of the Rx and Sr33 CC domains. The small-angle X-ray scattering (SAXS) data for MLA10 CC domain published by <cite class="ltx_cite raw v1">\cite{Casey2016}</cite> indicates the MLA10 CC domain also likely forms a four-helix bundle in solution, therefore a better approach to this experiment would be to generate a homology model of MLA10 based on the Sr33 CC domain four-helix bundle and then assess the effects of the mutants on protein stability using STRUM. Additionally, it would be ideal if all the approaches taken to the predictions in the manuscript could be detailed in the materials and methods.</li><li>To simulate protein stability, it would be more appropriate to use the entire functional region of the protein instead of a non-functional shorter fragment, as it is likely that the missing residues could contribute to stability of the protein. Unfortunately, there is no structure of the active CC domain of MLA10 (a minimum of 142 residues) and as STRUM requires a structure to predict destabilisation caused by point mutations, it is not possible to determine the destabilising effect of the Sr33 V20T and Sr33 G21E mutants in a functional CC domain, even via homology modelling. Consequently, any stability prediction using either the current MLA10 or Sr33 CC domain structures are essentially not directly relevant to the minimal functional domain.&nbsp;</li><li>Biochemical and biophysical analyses would be the more robust approaches than predictions of protein stability. These experiments are possible given that MLA10 CC domain can be purified in quantities sufficient for structural studies as in <cite class="ltx_cite raw v1">\cite{Maekawa2011}</cite>. For example, stability of MLA10 and Sr33 CC domain mutants could be analysed by circular dichroism (CD), 2D NMR, or by ThermoFluor. These experiments would be much more conclusive than prediction servers, and these data would significantly benefit the manuscript.</li><li>Finally, the MLA10 T20V and MLA10 E21G mutants appear to not have any effects on the cell death phenotype nor on the accumulation of these proteins in planta (Fig. 5). These observations are inconsistent with the hypothesis that the mutations destabilize the proteins. Further discussion of these observations would be beneficial.</li></ul><h2 data-label="485825" class="ltx_title_subsection"></h2><h2 data-label="485825" class="ltx_title_subsection"><b>(3)&nbsp;&nbsp;Other comments</b></h2><div>Figure S1 is a great positive control for the bioinformatics pipeline.</div><div></div><div>It is not clear why the second clade is described as a SUB-family of MLA given that ALL known MLA are in the other clade. The second clade is better described as MLA-like or MLA sister clade.</div><div></div><div>Some plants carry two (or even three) members of MLA. In these cases, do they belong to the same or a different subclade? It was not clear in the text and is worth commenting as this situation complicates allelic analyses.</div><div></div><div>Line 127: as there is not structural data available and to avoid confusion, the text should state “predicted to be located…”</div><div></div><div>Lines 142-143: The statement that RGH1/MLA family has been driven by subfamily-specific functionalization to distinct pathogens is highly speculative.&nbsp; Is there evidence for a second pathogen? Is possible that this subfamily detects uncharacterized powdery mildew strains. This contradicts lines 410-412 "Whether subfamily 2 NLRs confer disease resistance to avirulence genes present in yet uncharacterized Bgh populations or other pathogens remains to be tested".</div><div></div><div>Line 257: “Bootstrap not very high”. Perhaps include the bootstrap number in brackets.</div><div></div><div>Line 384. It is unclear how they can conclude from RNAseq data only that "in wild barley Rgh1/Mla has undergone frequent gene duplication (Table S1)". Could these sequences be allelic?</div><div></div><div>Lines 450-456. They cite <cite class="ltx_cite raw v1">\cite{Shen2007}</cite> as evidence that MLA-CC functions by binding WRKY transcription factors and derepressing them. Our understanding is that this model was drawn from an experiment with the inappropriate avirulence effector. </div><div></div><div>Figures 4 and 5: the loading control would be easier to distinguish when showing the band corresponding to RuBisCO (55KDa).&nbsp;<span class="math ltx_Math v1">\(\)</span></div><div></div><h1 data-label="924286" class="ltx_title_section">Reviewers</h1><div></div><div>Adam R. Bentham and Juan Carlos De la Concepcion, Department of Biological Chemistry, John Innes Centre, Norwich Research Park, Norwich, UK.</div><div></div><div>Sophien Kamoun. The Sainsbury Laboratory, Norwich Research Park, Norwich, UK.</div><div></div>