\documentclass[10pt]{article}

\usepackage{fullpage}
\usepackage{setspace}
\usepackage{parskip}
\usepackage{titlesec}
\usepackage{placeins}
\usepackage{xcolor}
\usepackage{breakcites}
\usepackage{lineno}





\PassOptionsToPackage{hyphens}{url}
\usepackage[colorlinks = true,
            linkcolor = blue,
            urlcolor  = blue,
            citecolor = blue,
            anchorcolor = blue]{hyperref}
\usepackage{etoolbox}
\makeatletter
\patchcmd\@combinedblfloats{\box\@outputbox}{\unvbox\@outputbox}{}{%
  \errmessage{\noexpand\@combinedblfloats could not be patched}%
}%
\makeatother


\usepackage[round]{natbib}
\let\cite\citep




\renewenvironment{abstract}
  {{\bfseries\noindent{\abstractname}\par\nobreak}\footnotesize}
  {\bigskip}

\renewenvironment{quote}
  {\begin{tabular}{|p{13cm}}}
  {\end{tabular}}

\titlespacing{\section}{0pt}{*3}{*1}
\titlespacing{\subsection}{0pt}{*2}{*0.5}
\titlespacing{\subsubsection}{0pt}{*1.5}{0pt}


\usepackage{authblk}


\usepackage{graphicx}
\usepackage[space]{grffile}
\usepackage{latexsym}
\usepackage{textcomp}
\usepackage{longtable}
\usepackage{tabulary}
\usepackage{booktabs,array,multirow}
\usepackage{amsfonts,amsmath,amssymb}
\providecommand\citet{\cite}
\providecommand\citep{\cite}
\providecommand\citealt{\cite}
% You can conditionalize code for latexml or normal latex using this.
\newif\iflatexml\latexmlfalse
\providecommand{\tightlist}{\setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}%

\AtBeginDocument{\DeclareGraphicsExtensions{.pdf,.PDF,.eps,.EPS,.png,.PNG,.tif,.TIF,.jpg,.JPG,.jpeg,.JPEG}}

\usepackage[utf8]{inputenc}
\usepackage[english]{babel}








\begin{document}

\title{UIUC Plant Physiology Journal Club: 2018-08-13~ ~}



\author[1]{Steven Burgess}%
\affil[1]{University of Illinois at Urbanaâ€“Champaign}%


\vspace{-1em}



  \date{\today}


\begingroup
\let\center\flushleft
\let\endcenter\endflushleft
\maketitle
\endgroup









\emph{Steven Burgess (0000-0003-2353-7794), Samuel Fernandes, Antony
Digrado, Charles Pignon, Elsa de Becker, Naomi Housego Day, Lusya
Manukyan, Stephanie Cullum, Isla Causon, Iulia Floristeanu, Young Cho,
Freya Way, Judy Savitskya, Robert Collison, Aoife Sweeney, Pietro
Hughes, Cindy Chan}

\subsection*{}

{\label{415290}}

\subsection*{Abstract}

{\label{415290}}

The paper ``Arabidopsis species employ distinct strategies to cope with
drought stress'' by Bouzid et al.
(\url{https://doi.org/10.1101/341859}\textbf{)} investigates whether
responses to water limitation vary between closely related species by
assessing the growth and survival of \emph{A. thaliana}, \emph{A.
lyrata} and \emph{A. halleri} accessions in a dry down experiment. By
including multiple accessions of each species the authors were able to
analyse variation in response to drought stress within and between
species based on eight phenotypic parameters. The authors went on to
perform comparative transcriptomic analysis between \emph{A. lyrata} and
\emph{A. halleri} over a time course of drought treatment and identified
differentially expressed genes. GO ontology analysis suggest the species
analysed adopt different strategies to cope with drought stress, with A.
lyrata employing avoidance and tolerance mechanisms, whereas \emph{A.
thaliana} showed strong avoidance but no tolerance. We were impressed
with the amount of work performed and thought the ~study aims to address
an interesting question. During the hour long journal club participants
were asked to focus on three aspects of the paper as part of a training
exercise, including novelty, interest, soundness as well as writing and
presentation.

\par\null

\subsection*{Review}

{\label{963733}}

There are several published papers looking at the effect of drought
stress in Arabidopsis species including \emph{A. lyrata} (Sletvold and
Agren 2011; Paccard et al. 2014) and A. thaliana (Ferguson et al. 2018;
Kalladan et al. 2017). We suggest toning down the assertion on Line 28
that little is known about the physiological response to drought in
closely related species. Although not in a single paper, this issue has
been addressed within a species by Sletvold and Agren (2012) and Davila
Olivas et al. (2017) and there is a fairly large collection of papers
looking at this within a species. To our knowledge, the novelty of this
work lies in analysis of drought responses within and between several
species of Arabidopsis in one article. We thought this was an
interesting approach and that the authors can make more of this point,
highlighting the new information that it yields.

\par\null

The article is well written in clear sentences and it was easy to read.
We felt the authors had collected a lot of data and believe this could
be explored further in the discussion, particularly the differential
expression data. There is a lot of microarray and transcriptomic data
available for the response of \emph{A. thaliana} to drought conditions
and would like to have seen some form of comparison between these data
and that collected for \emph{A. lyrata} and \emph{A. halleri}.

\par\null

In addition, it would help to provide more information about why
analysis of Arabidopsis accessions was limited to late flowering
varieties. Does excluding accessions which can terminate life cycle
early bias the experiment? Termination is a major strategy for survival
and may impacts upon the conclusion that ``response to depletion in SWC
did not reveal significant differences between accessions (line 591).''
Further discussion of this conclusion in the context of previous studies
would be illuminating as it appears to contrast with some findings, for
example Bouchabke et al. (2008) which suggested there are differences in
response to depletion of SWC between \emph{A. thaliana} accessions.
Readers would also benefit from discussion about how the results from
the phenotypic analysis relates to other studies which have implicated
trichome production, rosette leaf size and flowering time as drivers for
drought tolerance.

\par\null

We commend the fact the methods are detailed which should aid anyone
wanting to replicate the study. Several aspects were highlighted as
excellent practices, in particular: the fact that all program versions
are provided, software parameters are included and accessions and
materials are well catalogued. To build on this we recommend depositing
the data (particularly transcriptomic analysis) in a public repository
such as GEO, SRA or Zendo as required by some journals. This means data
can be built upon in future studies and could increase the likelihood of
the paper being cited. Inclusion of extended methods in an accompanying
Bioprotocol paper or on sites such as Protocols.io and sharing custom
scripts in a github repository (or on Zendo) would make the reporting of
methods outstanding.~

\par\null

\subsection*{Minor comments}

{\label{868571}}

The paper may benefit from clarify a number of questions raised by
participants: ~~

\begin{itemize}
\tightlist
\item
  How was soil mixture chosen?
\item
  Line 192 under what conditions were the plants grown in the
  greenhouse?
\item
  Line 206: How were the first signs of wilting defined? It might be
  worth mentioning this earlier in the manuscript (line 206)
\item
  Line 232 - How was loss of turgidity measured?
\item
  The colors used in the figures will create difficulty for those
  individuals with color blindness, using color oracle
  (\url{https://colororacle.org/}) can help address this issue.
\item
  Figure 2 - no scale bars are included
\item
  Figure 2 - how many plants per pot were analyzed? This could have
  impacted on measurements displayed images suggested this was variable.
\item
  Figure 5 - we thought it was excellent that unit level data is
  provided, this could be extended to the other figures too.
\item
  Figure legends could be improved e.g. by changing ``halleri'' to ``A.
  halleri''*Formating of legend (line 1326 )extra space and period.
\item
  The article might benefit from consistent formatting/presentation of
  figures
\item
  A table might be more appropriate for Figures 7 and 8
\item
  Inclusion of n= numbers in the figures would help readers assess the
  data.
\item
  We were confused about how the experiment was designed, why is data
  for only on biological replicate displayed in figures 7 and 8.
\end{itemize}

\par\null

\subsection*{References}

{\label{453131}}

Bouchabke et al. 2008 \url{https://doi.org/10.1371/journal.pone.0001705}

Davila Olivas et al. (2017)~\url{https://doi.org/10.1111/mec.14100}

Des Marais, et al. (2012) https://doi.org/10.1105/tpc.112.096180

Huttunen et al. 2010 \url{https://doi.org/10.5735/085.047.0304}

Kalladan et al. (2017)~\url{https://doi.org/10.1073/pnas.1705884114}

Paccard et al. (2014) doi: 10.1007/s00442-014-2932-8

Sletvold and Agren (2011)
\url{https://doi.org/10.1007/s10682-011-9502-x}

\selectlanguage{english}
\FloatBarrier
\end{document}

