<div>    <i>Steven Burgess (0000-0003-2353-7794), Samuel Fernandes, Antony Digrado, Charles Pignon, Elsa de Becker, Naomi Housego Day, Lusya Manukyan, Stephanie Cullum, Isla Causon, Iulia Floristeanu, Young Cho, Freya Way, Judy Savitskya, Robert Collison, Aoife Sweeney, Pietro Hughes, Cindy Chan</i></div><div>        </div><h1 data-label="752720" class="ltx_title_section">Abstract</h1><div>This review is compiled from notes taken by the UIUC Plant Physiology preprint journal club during a one hour session on 2018-07-30. The review refers to the preprint "Natural variation in stomata size contributes to the local adaptation of water-use efficiency in Arabidopsis thaliana" by Hannes Dittberner, Arthur Korte, Tabea Mettler-Altmann, Andreas Weber, Grey Monroe,<a href="http://orcid.org/0000-0002-2942-4750"> </a>and Juliette de Meaux ( doi: <a href="https://doi.org/10.1101/253021">https://doi.org/10.1101/253021</a>) published on BioRxiv.</div><div></div><h1 data-label="304309" class="ltx_title_section">Review</h1><div>In the paper “Natural variation in stomata size contributes to the local adaptation of Water use efficiency in Arabidopsis thaliana” Dittberner et al. (2018) investigated the genetic basis of water use efficiency using genome wide association analysis. Findings included (1) that there is substantial variation in stomatal size and patterning in A. thaliana accessions (2) decreased stomata size correlates with increased water use efficiency (3) two novel QTL affecting WUE independently of stomatal patterning (4) water use efficiency is a polygenic trait and (5) natural selection contributed to the establishment of variation in WUE in accordance with climatic conditions.</div><div></div><div>The findings are in accordance with previous studies looking at genetic variation in stomatal size and patterning in A. thaliana (Delgado et al. 2011; Monda et al. 2016), the role of natural selection in WUE of tomato (Muir et al. 2014), analysis of genomic variation in WUE in A. thaliana (Easlon et al. 2014; Aliniaeifard and van Meeteren 2014) and the extensive literature correlating a decrease in stomatal size with increased water use efficiency in many species.</div><div>We were impressed by the significant technical advance presented in the form of automated stomatal counting and suggest more could be made of this aspect of the paper. For example the we suggest inclusion of the term ‘high-throughput’ in the title, and think it would be helpful to include a picture of how the screening system works in the introduction.</div><div></div><div>Further, we found the conclusion that water use efficiency is a polygenic trait very interesting. The implications are important for studies aimed at the improvement of photosynthesis/WUE through conventional breeding or genetic manipulation. We wonder if there are other papers which have already discussed the potential polygenic nature of stomatal traits? Yoo et al. (2011) mentions different genes involved in WUE and stomatal patterning. This kind of paper could have been worth-mentioning in the discussion.</div><div></div><div>The identification of two novel QTL, not otherwise known to influence water use efficiency or to be related to stomatal patterning and regulation was particularly intriguing, and these two genes provide a great starting point for further analysis in future publications.</div><div></div><h2 data-label="289590" class="ltx_title_subsection">Major comments</h2><div>We believe the kinship matrix accounts for relatedness, and is usually called the k model. To account for population structure one could use principal components, combining these two approaches is known as the Q+K model. It would be helpful to include a citation for calculating the kinship matrix as there are multiple methods available. It might also be worth providing QQplots to show whether including population structure is necessary or not for this study.</div><div>We found the use of a Bonferroni correction in the analysis quite conservative, there are other options such as Benjamini-Hochberg adjusted p-values that could lead to declaring other SNPs as significant. Also, from the GWAS results, there seem to be other SNPs popping up - although, not significant. It might be worth using an alternative to p-values when investigating the possible biological significance of outstanding SNPs on a Manhattan plot - such as selecting the top 10 hits for investigation.</div><div></div><h2 data-label="728190" class="ltx_title_subsection">Minor comments:</h2><ul><li><div>Keywords: Consider revisiting, as many of the keywords are already on the title. Additionally, it would be helpful to include terms rather than acronyms which are not familiar to non-experts.</div></li><li><div>Introduction: Would be good to include examples of some of the limitations to automated confocal microscopy approaches to orientate readers.</div></li><li><div>Line 143: (Atwell et al., 2010). This citation doesn't seem to be right. If the author is referring to genomic heritability it could cite de los Campos et al. (2015).</div></li><li><div>Methods: Would like to see further information on the methodology used for carbon isotope discrimination in the main text</div></li><li><div>Results: Might consider using the term ‘genetic-heritability’ rather than ‘pseudo-heritability’</div></li><li><div>Results: Direct comparison with previously reported dC13 values for accessions that have been demonstrated to differing WUE would be helpful.</div></li><li><div>Figure 4: the text on the figure is likely to be too small to see properly</div></li></ul><ul><li><div>Line 505: “soil composition...” This is a really interesting point about the complexity of environmental effects on plant physiology. It might be helpful to provide some explanation for readers less familiar with soil content. (e.g. water holding content and effect on root structure - e.g.<a href="https://doi.org/10.1073/pnas.1721749115">https://doi.org/10.1073/pnas.1721749115</a>)</div></li><li><div>Line 499-500. Repetition of text on 498-499</div></li><li><div>Results: It would be helpful to share where the functional annotation for genes come from. This gene appears to be involved in mRNA decapping. There is only one publication, which found it is involved in PAMP triggered immunity, plants were dwarf in stature. TAIR does not annotate a role in cell differentiation (10.15252/embj.201488645)</div></li><li><div>Discussion: It is interesting to see a comparison with a related species and how the observed pattern is consistent between the two.</div></li></ul><ul><li><div>Data accessibility: This is excellent. Great to see that all data and code will be made available and deposited in a permanent repository!</div></li></ul><h2 data-label="424989" class="ltx_title_subsection">References</h2><div>Aliniaeifard and van Meeteren (2014) J Exp Bot. 65(22): 6529–6542.</div><div>de los Campos et al. (2015) PLoS Genet 11(5): e1005048. doi:10.1371/journal.pgen.1005048</div><div>Delgado et al. (2011) Ann Bot.; 107(8): 1247–1258</div><div>Easlon et al. (2014) Photosynth Res 119: 119. https://doi.org/10.1007/s11120-013-9891-5</div><div>Monda et al. (2016) Plant Physiol, 170: 1435–1444</div><div>Muir et al. (2016) Genetics. 2014 Dec; 198(4): 1629–1643.</div><div>    <b></b></div><div>    </div>